hibernate-core/src/test/java/org/hibernate/orm/test/mapping/converted/converter/onetoone/BidirectionalOneToOneWithConverterEagerTest.java

```diff
@@ -99,24 +91,18 @@ public void jdbcExecuteStatementStart() {
 
 	@Test
 	public void testBidirectionalFetchInverse(SessionFactoryScope scope) {
+		final SQLStatementInspector sqlStatementInterceptor = (SQLStatementInspector) scope.getStatementInspector();
```

Comment:
FYI, you can call the `getCollectingStatementInspector()` method, that way you can get rid of the cast ...

Reply:
thanks, gonna use that!